home/import_ordered_content_sets.py

```diff
@@ -133,24 +141,25 @@ def _add_pages(
         """
         ordered_set.pages = []
         for page in pages:
-            if not page.page_slug or page.page_slug == "-":
-                continue
-            content_page = ContentPage.objects.filter(slug=page.page_slug).first()
-            if page:
-                ordered_set.pages.append(
-                    (
-                        "pages",
-                        {
-                            "contentpage": content_page,
-                            "time": page.time or "",
-                            "unit": page.unit or "",
-                            "before_or_after": page.before_or_after or "",
-                            "contact_field": page.contact_field or "",
-                        },
+            if page.page_slug and page.page_slug != "-":
+                content_page = ContentPage.objects.filter(slug=page.page_slug).first()
+                if content_page:
+                    ordered_set.pages.append(
+                        (
+                            "pages",
+                            {
+                                "contentpage": content_page,
+                                "time": page.time or "",
+                                "unit": page.unit or "",
+                                "before_or_after": page.before_or_after or "",
+                                "contact_field": page.contact_field or "",
+                            },
+                        )
+                    )
```

Comment:
This is very verbose. Can we build the dict first and avoid most of the brackets-only lines?
```suggestion
                    os_page = {
                        "contentpage": content_page,
                        "time": page.time or "",
                        "unit": page.unit or "",
                        "before_or_after": page.before_or_after or "",
                        "contact_field": page.contact_field or "",
                    }
                    ordered_set.pages.append(("pages", os_page))
```